artifactory/services/buildinfo.go

```diff
@@ -66,3 +75,28 @@ func (bis *BuildInfoService) GetBuildInfo(params BuildInfoParams) (*buildinfo.Bu
 
 	return &buildInfoJson.BuildInfo, nil
 }
+
+func (bis *BuildInfoService) PublishBuildInfo(build *buildinfo.BuildInfo) error {
+	content, err := json.Marshal(build)
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+	if bis.IsDryRun() {
+		log.Output(clientutils.IndentJson(content))
+		return nil
+	}
+	httpClientsDetails := bis.GetArtifactoryDetails().CreateHttpClientDetails()
+	utils.SetContentType("application/vnd.org.jfrog.artifactory+json", &httpClientsDetails.Headers)
+	log.Info("Deploying build info...")
+	resp, body, err := bis.client.SendPut(bis.ArtDetails.GetUrl()+"api/build/", content, &httpClientsDetails)
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode != http.StatusNoContent {
```

Comment:
I think we should change it to:
```if resp.StatusCode != http.StatusCreated && resp.StatusCode != http.StatusNoContent {```
to support a future response code in Artifactory for this request.

Reply:
Added support for StatusCreated response code.